rest/route/task_restart.go

```diff
@@ -56,24 +56,24 @@ func (trh *taskRestartHandler) Parse(ctx context.Context, r *http.Request) error
 func (trh *taskRestartHandler) Run(ctx context.Context) gimlet.Responder {
 	err := resetTask(trh.taskId, trh.username)
 	if err != nil {
-		return gimlet.MakeJSONErrorResponder(err)
+		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "restarting task"))
```

Comment:
Probably don't need to wrap this since the helper function sets the errors messages.